.cargo/config.toml

```diff
@@ -8,7 +8,7 @@ rustflags = [
   "-C",
   # increase the stack size to prevent overflowing the
   # stack in debug when launching sub commands
-  "link-arg=/STACK:4194304",
+  "link-arg=/STACK:8388608",
```

Comment:
Having to do this after doing a bit of re-organization in the factory code seems very strange. It would be nice if we could get to the bottom of this because I don't believe the factory code should be storing that much data.

Reply:
The size of our main future bugs me a bit.